session/object.go

```diff
@@ -266,7 +271,7 @@ func (x Object) AssertObject(obj oid.ID) bool {
 }
 
 // ObjectVerb enumerates object operations.
-type ObjectVerb int8
+type ObjectVerb int32
```

Comment:
what is a practical motivation for this change? we are not ready for MaxInt32 number of object operations for sure, it will not be possible to use

Reply:
previously we made `int8(int32)` cast w/o any precondition, and this was obviously incorrect. I considered two options:
1. limit possible value by 127
2. just move to `int32`

with 1st, type transition wont be needed. But i though that such limit - although as you noted in practice it is quite legit - would be pretty synthetic. Yeah we definitely wont go over 127 real values. However, given that this is a protocol field, sometimes there is a need to reserve extreme values for some specifics, for example max int32 in this case. While value space extension is a backward compatible change, this would require type extension anyway. So, in order not to complicate things, I simply expanded the type to the gRPC